LondonBritishLibrary/orient/BLorient795.xml

```diff
@@ -32,8 +35,804 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                       <layoutDesc>
+                           <layout columns="3" writtenLines="31 32">
+                           </layout>
+                       </layoutDesc>
+                   </objectDesc>
+                   <handDesc>
+                       <handNote script="Ethiopic" xml:id="h2">
+                           <desc>Written in a good hand.</desc>
+                           <date notBefore="1730" notAfter="1755"/>
+                       </handNote>
+                   </handDesc>
+                   <additions>
+                       <list>
+
+                         <item xml:id="a1">
+                           <locus target="#2r"></locus>
+                           <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                               <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                           <q xml:lang="gez">መጽሐፈ፡ ኪዳን፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                     </q>
+                         </item>
+                         <item xml:id="a2">
+                           <locus target="#24r"></locus>
+                           <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                               <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                           <q xml:lang="gez">ሲኖዶስ፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                     </q>
+                         </item>
+                       </list>
+                       <list>
+                         <item xml:id="e1">
+                           <desc>The names of the owner <persName ref="PRS13173WaldaK" role="owner"></persName>
+                         and <persName ref="PRS13174Gorgoryos" role="scribe"></persName> are mentioned
+                       on <locus target="#73r #122r"></locus>.</desc>
+                         </item>
+                       </list>
+                   </additions>
+               </physDesc>
+               <history>
+                   <origin>
+                       <origDate notBefore="1730" notAfter="1755" evidence="reign">
+                         The name of <persName ref="PRS5617IyasuII"></persName> is mentioned
+                         in the prayer on <locus target="#44v"></locus>.
+                       </origDate>
+                   </origin>
+               </history>
+               </msPart>
+               <msPart xml:id="p3">
+                 <msIdentifier></msIdentifier>
+                 <msContents>
+                   <msItem xml:id="p3_i1">
+                     <locus from="123r"></locus>
+                     <title type="incomplete" xml:lang="en">Theological work</title>
+                     <textLang mainLang="gez"></textLang>
+                     <note>The work was left unfinished by the scribe.</note>
+                     <note>The rubrics are partly lacking.</note>
```

Comment:
See above - if we assume that it's about rubrication, I would say that the extra is actually enough